packages/react-liveness/src/components/FaceLivenessDetector/service/utils/blazefaceFaceDetection.ts

```diff
@@ -1,5 +1,11 @@
+// TODO Thadd fix this
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
 import * as tf from '@tensorflow/tfjs-core';
-import * as blazeface from '@tensorflow-models/blazeface';
+import * as faceDetection from '@tensorflow-models/face-detection';
```

Comment:
Any chance we don't need to import * all of blazeface in the new version? Is this still the recommendation? Would love to cut down on bundle size if it's possible to be more sepcific

Reply:
Will test it out!